### DISCUSSION TITLE
Custom CSS theme - Pure BLACK / album color, Rounded off Corners, Font Change

### DISCUSSION THREAD

**Discussion by ht-suji** [pear-devs/pear-desktop#1346] [Show and tell]
I make a custom CSS theme. cause i Really wanted to change the default font :) (and since it doesn't have a plugin)

## Looks:
myStyles-Abril_Fatface,PoppinsDark1:
![myStyles-Abril_Fatface,PoppinsDark1](https://github.com/th-ch/youtube-music/assets/132142130/701b80df-b7a5-457c-97a8-7b1999eccabc)
myStyles-Nunito,PoppinsAlbum1:
![myStyles-Nunito,PoppinsAlbum1](https://github.com/th-ch/youtube-music/assets/132142130/9ebb0624-f252-4bf8-a9c4-75f33b709ac5)
myStyles-Nunito,PoppinsAlbum2:
![myStyles-Nunito,PoppinsAlbum2](https://github.com/th-ch/youtube-music/assets/132142130/48f5fa9b-20df-4cae-8196-0ba0cc623a0c)
myStyles-Nunito,PoppinsDark1:
![myStyles-Nunito,PoppinsDark1](https://github.com/th-ch/youtube-music/assets/132142130/546650f2-2e2e-476b-8d3f-25014b8a0cbf)
myStyles-Nunito,PoppinsDark2:
![myStyles-Nunito,PoppinsDark2](https://github.com/th-ch/youtube-music/assets/132142130/95f67d21-adf8-4f9b-b374-5b22a87e47a2)

### Changes: 

**Apr 04 2024**
- added Album theme css which blends the whole color of the app when using album theme plugin
- added preview photos

**Oct 23 2023**
- Pure Black for the bottom player (album-color-theme should be off for it to be all black) 
- rounded off corners
- Font Change (Headings and Normal) (can be changed in CSS file), 
  - Local fonts (Installed on pc) works fine
  - online fonts (sort of) works 

> I used this site for good pairing fonts [https://www.w3schools.com/css/css_font_pairings.asp](url)

### Steps:

To add it, 

1. Copy the CSS code pinned at the last,
1. Save it in `.css` format.
1. Open the app,
1. Options -> Visual Tweaks -> Themes -> Import Custom CSS. and select this file.
1. Restart App.
(It will load from the location you saved it, each time when opening so don't delete the CSS file)

Make any changes that is appropriate , this is my first time writing a CSS file so i pretty much just used  `!important` keyword for every change i made.

### custom fonts:

for some reason i can't use the `@import url("")`, it's just not loading it but, i can copy the content from the url and paste it in the css, then it works, so **_sort of_** can load online fonts.
copy the contents from here - `https://fonts.googleapis.com/css?family=<font1>|<font2>` change font1 and font2 as the font u like in google fonts

For Example,
`@import url("https://fonts.googleapis.com/css?family=Abril+Fatface|Poppins")` doesn't work
i instead copy the content from that url into the css - then it works.

Change the variables if you want.

> ```css
> /*VARIABLES*/
> :root {
>   --radius_border: 7px;
>   --header_font: "Albril Fatface"; 
>   --ui_font: "Poppins";>   
> }
> ``` 

my themes:
- myStyles-Abril_Fatface,NunitoDark.css
- myStyles-Abril_Fatface,PoppinsDark.css
- myStyles-Nunito,PoppinsAlbum.css
- myStyles-Nunito,PoppinsDark.css

[YTMusicThemes.zip](https://github.com/th-ch/youtube-music/files/14857147/YTMusicThemes.zip)

**Comment by pollux78** [comment]
Had to find this old discussion thread to solve this issue myself, thank you!

**Comment by Pableske** [comment]
Hello, when using this theme the app becomes slow when in the playlist menu. Is this normal, or do I have something misconfigured?
